lib/wicked/IPv6.pm

```diff
@@ -0,0 +1,151 @@
+# SUSE's openQA tests
```

Comment:
generally I agree with idea of split our wickedbase which is really getting to big . but tricky part that currently you name file as IPv6 but in a fact it is not "list of IPv6 related  functions" **but** "list of functions which was used during development of IPv6 tests". Which is not the same and might create confusions in future . I mean that for example `evaluate_step` is not even wicked related and maybe better to be in `lib/utils.pm` after all . Also it looks strange to have `obtain_address_by_dhcpv4 ` within IPv6 module . 

let's discuss this also on Monday :)

Reply:
I would avoid moving this very specific function to utils.pm, really. It would open more questions than answers.